server/metrics/metrics.go

```diff
@@ -2864,30 +2869,51 @@ var (
 	})
 
 	// ## Cache Proxy metrics
-	ByteStreamProxyReads = promauto.NewCounterVec(prometheus.CounterOpts{
+	ByteStreamProxiedRequests = promauto.NewCounterVec(prometheus.CounterOpts{
 		Namespace: bbNamespace,
 		Subsystem: "proxy",
-		Name:      "byte_stream_reads",
-		Help:      "The result of serving a byte_stream_proxy.read request out of the byte_stream_server_proxy.",
+		Name:      "byte_stream_requests",
+		Help:      "The number of requests served by a ByteStreamServerProxy by gRPC and cache hit/miss status.",
 	}, []string{
+		StatusLabel,
+		CacheHitMissStatus,
+	})
+	ByteStreamProxiedBytes = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: bbNamespace,
+		Subsystem: "proxy",
+		Name:      "byte_stream_bytes",
```

Comment:
Is this sort of ambiguous since it could be bytes read or written? would it make sense to include that in the metric name?

Reply:
Given that writes are always written to the remote cache, I think this name is fine. If you feel strongly or it becomes ambiguous/problematic I can change it.